r-route-table.tf

```diff
@@ -3,7 +3,7 @@ resource "azurerm_route_table" "route_table" {
   location            = var.location
   resource_group_name = var.resource_group_name
 
-  disable_bgp_route_propagation = var.disable_bgp_route_propagation
+  bgp_route_propagation_enabled = var.disable_bgp_route_propagation
```

Comment:
It is better to rename the variable to maintain consistency.

Reply:
Done. Please, if you could confirm if I used the right type for the commit.

Reply:
instead of `refactor!` you may want to use `feat(refactor):`

the ! means "breaking", and our release bot will want to tag a v8 (new major)